storm-core/src/storm.thrift

```diff
@@ -204,6 +203,7 @@ struct ExecutorStats {
   2: required map<string, map<string, i64>> transferred;
   3: required ExecutorSpecificStats specific;
   4: required double rate;
+  5: required map<string, map<string, double>> throughput;
```

Comment:
If we want to be able to add this as a rolling upgrade this needs to be optional, and we need the code everywhere to be able to handle when it is not provided.

To me this feature is not important enough to force downtime in a cluster.